bin/automatic_review.py

```diff
@@ -0,0 +1,169 @@
+        return_failed_metrics = ['fail', failed_metrics]
+    return return_failed_metrics
+
+
+class AutomaticRunReview():
+    def __init__(self, run_name, run_element_by_lane):
+        self.run_name = run_name
+        self.run_element_by_lane = run_element_by_lane
+
+    def patch_entry(self):
+        run_metrics = {}
+        for lane in self.run_element_by_lane:
+            lane_id = lane['lane_number']
+            run_metrics[lane_id] = metrics(lane, 'run')
+        if run_metrics:
+            for lane in run_metrics:
+                patch_review = {}
+                patch_comments = {}
+                review = run_metrics[lane][0]
+                if review == 'pass':
+                    patch_review['reviewed'] = 'pass'
+                    rest_communication.patch_entries('run_elements', payload=patch_review, update_lists=None, where={"run_id":self.run_name, "lane": lane})
+                elif review == 'fail':
+                    review_comments = ('lane ' + str(lane) + ' failed due to ' + ' '.join(run_metrics[lane][1]))
+                    patch_review['reviewed'] = 'fail'
+                    patch_comments[ELEMENT_REVIEW_COMMENTS] = review_comments
+                    rest_communication.patch_entries('run_elements', payload=patch_review, update_lists=None, where={"run_id":self.run_name, "lane": (lane)})
+                    rest_communication.patch_entries('run_elements', payload=patch_comments, update_lists=None, where={"run_id":self.run_name, "lane": (lane)})
+
+
+
+class AutomaticSampleReview():
+    def __init__(self, sample, sample_name):
+        self.sample = sample
+        self.sample_name = sample_name
+
+    def patch_entry(self):
+        sample_review = metrics(self.sample, 'sample')
+        review = ''
+        if sample_review:
+            if sample_review == ['pass']:
+                review = sample_review
+            else:
+                review = (sample_review[1])
+            patch_review = {}
+            patch_comments = {}
+
+            if review == 'pass':
+                patch_review['reviewed'] = 'pass'
+                rest_communication.patch_entries('samples', payload=patch_review, update_lists=None, where={"sample_id": self.sample_name})
+
+            else:
+                patch_review['reviewed'] = 'fail'
+                patch_comments[ELEMENT_REVIEW_COMMENTS] = 'sample ' + self.sample_name + ' failed due to ' + ', '.join(review)
+                rest_communication.patch_entries('samples', payload=patch_review, update_lists=None, where={"sample_id": self.sample_name})
+                rest_communication.patch_entries('samples', payload=patch_comments, update_lists=None, where={"sample_id": self.sample_name})
+
+def get_reviewable_runs():
+    runs = rest_communication.get_documents('aggregate/all_runs', depaginate=True, match={"proc_status":"finished","review_statuses":"not%20reviewed"})
```

Comment:
Could you check the latest version of rest_communication because I think you don't need ```depaginate=True``` but you might need ```pagination=False```